packages/teams-js/src/public/interfaces.ts

```diff
@@ -172,7 +172,12 @@ export enum FileOpenPreference {
 /**
  * @deprecated
  * As of 2.0.0, please use {@link app.Context} instead.
- * Represents structure of the received context message.
+ *
+ * @remarks
+ * For more details on the updated {@link app.Context} interface, visit
+ * {@link https://docs.microsoft.com/microsoftteams/platform/tabs/how-to/using-teams-client-sdk}.
+ *
```

Comment:
Could we add this anchor to the link: **#updates-to-the-context-interface**:

https://docs.microsoft.com/microsoftteams/platform/tabs/how-to/using-teams-client-sdk#updates-to-the-context-interface

. . . it will not resolve yet, but will work upon docs publish. (And browsers will ignore the broken anchor until then anyway).

Reply:
Great suggestion, thanks for the updated link. Added!